pkgs/development/python-modules/rkm-codes/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytest-runner
+}:
+
+buildPythonPackage rec {
+  pname = "rkm-codes";
+  version = "0.5";
+
+  src = fetchFromGitHub {
+    owner = "KenKundert";
+    repo = "rkm_codes";
+    rev = "v${version}";
+    sha256 = "sha256-r4F72iHxH7BoPtgYm1RD6BeSZszKRrpeBQccmT4wzuw=";
+  };
+  buildInputs = [ pytest-runner ];
+
+  # this has a circular dependency on quantiphy
+  preBuild = ''
+    sed -i '/quantiphy/d' ./setup.py
+  '';
+
+  # tests require quantiphy import
+  doCheck = false;
+
+  meta = with lib; {
+    description = "QuantiPhy support for RKM codes";
+    homepage = "https://nurdletech.com/linux-utilities/rkm_codes";
+    license = licenses.gpl3;
```

```suggestion
    license = licenses.gpl3Plus;
```

See https://github.com/KenKundert/rkm_codes/blob/master/pyproject.toml#L14